docs/admin/kubeadm.md

```diff
@@ -323,16 +323,60 @@ kind: NodeConfiguration
 caCertPath: <path|string>
 discoveryFile: <path|string>
 discoveryToken: <string>
-
-# Currently only the first server is used as a target for the cluster
-# bootstrap flow.
 discoveryTokenAPIServers:
-  - <address|string>
-  - <address|string>
-
+- <address|string>
+- <address|string>
 tlsBootstrapToken: <string>
 ```
 
+## Securing your installation even more
+
+Most security features are automatically turned on for kubeadm clusters, but there is a balance between usability
+and security. kubeadm clusters start with RBAC and the Node Authorizer turned on, and all communication between
+components secured for instance.
+
+### Turning off auto-approval of Node Client Certificates
+
+However, in order to make the `kubeadm join` flow smooth, there is an CSR auto-approver
```

Comment:
The `However` here is awkward as it refers to the previous section.  I'd instead say something in the previous paragraph like: "The defaults for kubeadm may not work for everyone.  This section documents how to tighten up a kubeadm install at the cost of some usability."